### PR TITLE
Limit the number of statements in exception raising test blocks to 1.

### PR DESCRIPTION
**Pull request by autoih**
We may locate the exact statement where the exception is raiased.

### CODE REVIEW

File: tensorflow/python/feature_column/sequence_feature_column_test.py [hunk 1]
```diff
@@ -466,9 +466,8 @@ def test_sequence_length_not_equal(self):
 
     with self.assertRaisesRegexp(
         errors.InvalidArgumentError, r'Condition x == y did not hold.*'):
-      _, sequence_length = sequence_input_layer({
-          'aaa': sparse_input_a, 'bbb': sparse_input_b})
-      self.evaluate(sequence_length)
+        _, sequence_length = sequence_input_layer({'aaa': sparse_input_a, 'bbb': sparse_input_b})
+        self.evaluate(sequence_length)
```
Comment (karmel) [inline tensorflow/python/feature_column/sequence_feature_column_test.py]: This seems like the wrong indenting-- revert this one?

**Review by karmel — CHANGES_REQUESTED** [review]
Thanks-- this looks mostly fine.

File: tensorflow/python/feature_column/sequence_feature_column_test.py (hunk 1 above)
Comment (I-Hong) [reply tensorflow/python/feature_column/sequence_feature_column_test.py]: Thanks @karmel. Updated.

**Comment by rthadur** [comment]
@karmel can you please review new changes ?

File: tensorflow/python/feature_column/sequence_feature_column_test.py [hunk 2]
```diff
@@ -464,10 +464,8 @@ def test_sequence_length_not_equal(self):
     sequence_input_layer = sfc.SequenceFeatures(
         [numeric_column_a, numeric_column_b])
 
-    with self.assertRaisesRegexp(
-        errors.InvalidArgumentError, r'Condition x == y did not hold.*'):
-      _, sequence_length = sequence_input_layer({
-          'aaa': sparse_input_a, 'bbb': sparse_input_b})
+    with self.assertRaisesRegexp(errors.InvalidArgumentError, r'Condition x == y did not hold.*'):
+      _, sequence_length = sequence_input_layer({'aaa': sparse_input_a, 'bbb': sparse_input_b})
```
Comment (karmel) [inline tensorflow/python/feature_column/sequence_feature_column_test.py]: These lines look too long.

**Review by karmel — DISMISSED** [review]
Looks like there are some conflicts + lint errors that need to be resolved, but once tests are passing, LGTM.

File: tensorflow/python/feature_column/sequence_feature_column_test.py (hunk 2 above)
Comment (autoih) [reply tensorflow/python/feature_column/sequence_feature_column_test.py]: Thanks @karmel, fixed.

**Comment by rthadur** [comment]
@autoih thank you , there are still conflicts to resolve.

**Comment by autoih** [comment]
I've tried to resolve the conflicts in Line `669-672` and `694-697` (remove them), but once I removed them, CLA issue raises up.

**Comment by rthadur** [comment]
@autoih are there any other contributors to this PR , if yes they need to sign CLA.

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Pull request closed without merging**